tests/test_database.py

```diff
@@ -0,0 +1,27 @@
+
+import unittest
+from app.dojo import *
+
+
+class TestDatabase(unittest.TestCase):
+    def setUp(self):
+        self.dojo = Dojo()
+
+    def test_successfully_saves_state(self):
+        """ Test saves application state to Database. """
+        self.dojo.create_room('living', ["kigali"])
+        self.dojo.create_room('office', ["Django"])
+        self.dojo.add_person("Maina", "oscar", "fellow")
+        output = self.dojo.save_state("somedb")
+        self.assertEqual("App data has been stored in somedb.db database", output)
```

Comment:
Another thing, the format for self.assert is (output, expected output). Just revise that.